Drivers/rc_receiver/Src/sbus_receiver.cpp

```diff
@@ -12,6 +12,7 @@ SBUSReceiver::SBUSReceiver(UART_HandleTypeDef* uart) : uart_(uart)
     received_sbus_.failsafe = false;
     received_sbus_.lost_frame = false;
     received_sbus_.new_data = false;
+    this->setIsDataNew(false);
```

Comment:
perhaps you can just use the initializer list to set the value of `is_data_new_`